pymc_marketing/mlflow.py

```diff
@@ -548,12 +950,15 @@ def new_sample(*args, **kwargs):
                     **arviz_summary_kwargs,
                 )
 
-            model = pm.modelcontext(kwargs.get("model"))
             if log_model_info:
                 log_model_derived_info(model)
 
-            if log_datasets:
-                log_data(model=model, idata=idata)
+            if log_metadata_info:
+                log_metadata(model=model, idata=idata)
+
+            # mmm.fit() calls pm.sample() internally, so we need to make sure log-likelihood is only added once
+            if log_loocv and "log_likelihood" not in idata.groups():
+                pm.compute_log_likelihood(idata=idata, model=model)
```

Comment:
Think this is fine. I think we should ideally have it separate though and expose an MMM method. Let's hold off on that in this PR